### ISSUE TITLE
mypy failing with "config not defined"

### ISSUE THREAD

**Issue by sanders41** [ethyca/fidesops#449]
### Bug Description

mypy is failing with the following:

```console
src/fidesops/service/connectors/saas_connector.py:98: error: Name 'config' is not defined  [name-defined]
                if config.dev_mode:  # pylint: disable=R1720
                   ^
src/fidesops/service/connectors/saas_connector.py:129: error: Name 'config' is not defined  [name-defined]
        if config.dev_mode:
           ^
Found 2 errors in 1 file (checked 146 source files)
```

I did a bisect and it started in this commit https://github.com/ethyca/fidesops/commit/12e25a175526f7c70625cf62f604ffab7bd2088d (happens to be the most recent).

#### Steps to Reproduce

1. Run `make mypy`
2. Running mypy outside make `mypy src --ignore-missing-imports` gets the same error.

### Expected behavior

No mypy errors

### Screenshots

N/A

### Environment

* Version:
* OS: Linux
* Python Version: 3.10
* Docker Version: 20.10.14

### Additional context

N/A

**Comment by sanders41** [comment]
It looks like #448 will fix this. This brings up another issue. Both MyPy and Pylint should have failed CI from this, but it looks like both passed. On first pass I don't see any reason for this.

**Comment by pattisdr** [comment]
I think https://github.com/ethyca/fidesops/pull/363 would have failed if we had gotten it up-to-date with main before merging. It was an old PR that had been sitting for awhile.

**Comment by sanders41** [comment]
Closed via #448
